index.js

```diff
@@ -154,10 +156,6 @@ Sevnup.prototype._getOwnedVNodes = function _getOwnedVNodes() {
 
 Sevnup.prototype._onRingStateChange = function _onRingStateChange() {
     var self = this;
-    if (!this.running) {
```

Comment:
What's the reasoning for removing this?

Reply:
Just refactoring to bring coverage back to 100%